test/common_test/world_features.cc

```diff
@@ -16,6 +16,8 @@
 */
 #include <gtest/gtest.h>
 
+#include <LinearMath/btScalar.h>
```

Comment:
I think we don't want to include a bullet header file in a common test

Reply:
oh, I see it's in order to use the `btGetVersion()` API. I think I would prefer detecting the version in cmake and use `target_compile_definitions` to define a macro indicating whether the version is `>= 2.89` or `< 2.89`

Reply:
removed header and updated other places in test dir that use `btGetVersion()`. It looks like our `FindGzBullet` does not set a `GzBullet_VERSION` variable or `Bullet_VERSION`. So I added extra logic in the test cmake file to get the bullet version using `pkg_check_module`. We can also move that logic to gz-cmake later.

Reply:
nit: my favorite way to do this is to do the version check in cmake with its [version comparisons](https://cmake.org/cmake/help/latest/command/if.html#version-comparisons) (like `VERSION_GREATER_EQUAL`), and then use version comparison in the macro name, like `BT_BULLET_VERSION_GE_289` (see an [example from gz-common](https://github.com/gazebosim/gz-common/blob/983a1eb4bec2753c7c1313534c99d26474a3a9c4/graphics/src/CMakeLists.txt#L79-L85)). This is a little more robust if there was a patch version added for example

Reply:
updated checks, added definitions for `BT_BULLET_VERSION_LE_325` and `BT_BULLET_VERSION_GE_289` [b963af1](https://github.com/gazebosim/gz-physics/pull/574/commits/b963af17b666bcbe5c8ea10a16f6cbe07d8ad651)`